iis_action.ps1

```diff
@@ -0,0 +1,59 @@
+Param(
+    [parameter(Mandatory = $true)]
+    [ValidateSet( 'start', 'stop', 'restart')]
```

Comment:
Is this case sensitive?

Reply:
I'll just force the input to lower case before giving it to the action.